### PR TITLE
Apple clone

### PR DESCRIPTION
**Pull request by evansinho**
# Apple clone
### the project involves using :
- A background image
- A gradient
- A navigation bar with links
- A footer

### CODE REVIEW

**Comment by uryelah** [comment]
Hello **Evanson Igiri**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all, great job applying the gradient style, your navbar looks awesome!🌟**

I see that you closed the first PR to add stickler and that you've made some changes requested by the first TSE already. Your project is almost there but still needs a couple of required changes, hang in there! 💪

---------    

## Required Changes 🏗 👷‍♀ 

- [ ] Make sure there are no errors or warnings in your HTML validation. [Use this validator.
](https://validator.w3.org/nu/)
- [ ] The bottom images should be wrapped inside `<a>` tags.

- [ ] The bottom images should be horizontally aligned with the main image. Check the screencaps bellow comparing your clone and the original site:
**Clone:**
Screenshot_2020-04-06 Apple
**Original:**
Screenshot_2020-04-06 Apple(1)

If this is not apparent in your browser due to screen size differences you can press 'Ctrl' + '-' keys on your keyboard for a quick check of how the page looks in slightly wider screens.

----

- [ ] Make sure the font-weight and family of the footer text are close to the original ones.

-----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by uryelah** [comment]
Hey Evanson, good job fixing th evalidator errors and adding a tags, make sure to apply all the required changes before asking for a new code review though.

Notice the empty space in the right highlighted in the screencap below.
Make sure the main image takes 100% of the screen width and the bottom images section takes the same width.
Screenshot_2020-04-06 Apple


## Required Changes 🏗 👷‍♀ 

- [ ] The bottom images should be horizontally aligned with the main image. Check the screencaps bellow comparing your clone and the original site:

----

- [ ] Make sure the font-weight and family of the footer text are close to the original ones.

-----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by evansinho** [comment]
Hello @uryelah, Thanks for the quick response,
I Have made the changes and it looks good on my system even after increasing my screen size, it still looks good

**Comment by uryelah** [comment]
Hey Evanson, it still looks the same as the screencap from the last code review. 

To fix this you can use `margin: 0 auto` on the header and `justify-content: center` on the bottom images, or similar. 

This project is not about responsive design, but the site should look the same on any screen size. 
Try viewing it on a wider screen with [your browser responsive tools.](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode)

If you need any further assistance feel free to reach out on slack to @sarah.

**Comment by uryelah** [comment]
Hello Evanson, please make sure you read carefully the implementation suggestions in the last code review and make sure the project looks centered and taking full-screen width on chrome and firefox.

The issue from the last reviews still remains, if you are having issues with the fix please feel free to reach out via slack.

**Review by serdg0 — CHANGES_REQUESTED** [review]
**Status: Require Changes.**
Hello Evanson! :v:
You are really closing to finish this :muscle:
Only this issue, as adressed before: 

- [ ] There is stil a gap at 1040px screen. You can even notice the horizontal scroll bar. Take a look:

Screenshot from 2020-04-06 09-12-11

Remember, this projects will be seen by future potential employers :moneybag: :moneybag: 
So if you have any doubt, don't hesitate to reach me on slack (serdg)

### Happy Codding! :computer:

**Comment by evansinho** [comment]
hey @serdg0 could you please check again because I don't see the issue on my end even after adjusting the screen size it's still evenly aligned

File: style.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+header {
+  background: url(images/your_verse_hero.jpg) 50% 100% no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 535px;
+  margin: 0 auto;
+  max-width: 100%;
+}
```
Comment (men32z) [inline style.css]: max-width should be a pixel size, and if you want percentages you should put it in a with property, this way you have your responsive width with the boundaries that you want.
```suggestion
header {
  background: url(images/your_verse_hero.jpg) 50% 100% no-repeat;
  display: flex;
  flex-direction: column;
  align-items: center;
  height: 535px;
  margin: 0 auto;
  max-width: 1440px;
  width: 100%;
}
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+/* grid-section */
+
+.grid-image {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  padding: 5px 0 10px 0;
+}
```
Comment (men32z) [inline style.css]: here is almost the same, you already have a width 100% but you want it to be the same size as the other one in wide screens, so you set the limit in 1440px which is the original size and keep it responsive with that 100% in width property. and don't forget to add the "margin: 0 auto" to center the dive as the header.

```suggestion
.grid-image {
  display: flex;
  justify-content: center;
  width: 100%;
  padding: 5px 0 10px 0;
  max-width: 1440px;
  margin: 0 auto;
}
```

**Review by men32z — CHANGES_REQUESTED** [review]
Hey @evansinho !
## Good job on this project! 👍

I see that you have problems to see the alignment, I recommend you to use  both, Firefox and Chrome because sometimes properties don't work as expected in all browsers,  
take a look on this images.

Screenshot_2020-04-06 Apple(1)
Screenshot_2020-04-06 Apple

this is the general problem that we have, the main image should be aligned with the bottom images, if you can't see this in your computer try to use a bigger monitor or do "ctrl + minus" to adjust the size of the page and make it smaller to see it.  I additional made a review so you can see in wich part you have to focus. 
### task 
- [ ] fix the alignment problem pointed by the previous reviews.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! 👩‍💻

Happy coding 🤓

--Luis Preza

There are still some changes 👇 you need to implement to approve this request 🔍

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

Nice Job @evansinho in following the recommendations of previous reviewers.🎉🎉

This project meets the requirements for approval.

**STATUS: APPROVED ✅**

However kindly do this before merging:👇
- [ ] Kindly remove all inline code comments in your CSS file. If you need to check previous versions of your code use git history.

**I trust you would fix this before merging.👆**

Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**